HoI4Mod/Rome_Strikes_Back/events/tesb_ETH.txt

```diff
@@ -0,0 +1,2299 @@
+add_namespace = tesb_ETH
+
+
+### Invite Niger to faction events
+# Invite Niger to faction
+country_event = {
+	id = tesb_ETH.10
+	title = tesb_ETH.10.t
+	desc = tesb_ETH.10.d
+	picture = GFX_report_event_ger_civil_war
+
+	is_triggered_only = yes
+	# accept unconditionally
+	option = {
+		name = tesb_ETH.10.a
+		ETH = { country_event = { id = tesb_ETH.11 } }
+		trigger = {
+			NOT = { has_war_with = ETH }
+			is_in_faction = no
+		}
+		ai_chance = {
+			base = 100
+			modifier = {
+				factor = 0
+				NOT = { has_government = democratic }
+			}
+		}
+	}
+	# refuse
+	option = {
+		name = tesb_ETH.10.b
+		ETH = { country_event = { id = tesb_ETH.12 } }
+		ai_chance = {
+			base = 100
+			modifier = {
+				factor = 0
+				has_government = democratic
+			}
+		}
+	}
+}
+
+# Niger accepts
+country_event = {
+	id = tesb_ETH.11
+	title = tesb_ETH.11.t
+	desc = tesb_ETH.11.d
+	picture = GFX_report_event_ger_civil_war
+
+	is_triggered_only = yes
+	option = {
+		name = tesb_ETH.11.a
+		add_to_faction = NGR

```

Comment:
Changing from NGR to FROM will make this work for all of the "simple" join events. You can also use [From.GetName] (or GetNameDef and similar variations) to localize the event name/desc, etc.